### ISSUE TITLE
SocketIoSocket.send(): should include namespace in packet

### ISSUE THREAD

**Issue by DominicWatson** [trinopoty/socket.io-server-java#10]
Testing scenarios where using a non-default namespace showed that `socket.send( ... )` messages were not making it through to the client (or making it through to the browser, but not processed by the js client listeners).

Modifying `SocketIoSocket.java$send()` to the below resolved for me:

```java
sendPacket(packet);
// instead of
//  mClient.sendPacket(packet);
```

**Comment by trinopoty** [comment]
I use non-default namespaces in my projects and have yet to run across this issue. Let me check and get back to you.

**Comment by DominicWatson** [comment]
Using 2.3.1 client, i.e.

```<script src="https://cdn.socket.io/socket.io-2.3.1.js"></script>```

**Comment by trinopoty** [comment]
Sorry for the delay.
I've released a new version `1.0.3`. Should be available on maven in a few hours.

**Comment by DominicWatson** [comment]
Just seen as updating libs. belated thanks @trinopoty
